### PR TITLE
Added travis-ci and fixed coding standards issues

### PR DESCRIPTION
**Pull request by christianselig**
Added travis-ci to the module and fixed coding standards issues. PHP 5.4 is temporarily disabled in the .travis.yml script to prevent errors that need to be fixed in the future.

### CODE REVIEW

File: Form.inc [hunk 1]
```diff
@@ -2,10 +2,7 @@
 
 /**
  * @file
- *
- * Defines a class that allows drupal forms to be built via objects instead of arrays. Note that is still possible to
- * build non functional drupal forms using this class, the same rules apply that would normally apply to a Drupal API
- * Form.
+ * Defines a class that allows drupal forms to be built via objects instead of arrays. Note that is still possible to build non functional drupal forms using this class, the same rules apply that would normally apply to a Drupal API Form.
```
Comment (jonathangreen) [inline Form.inc:5]: Why did we squash this down to one line? Doesn't that break the 80 character rule? 

I believe according to the coding standards:
http://drupal.org/coding-standards/docs#file

We could use the first sentence as the summary, and then have the rest of the text as the additional documentation.

File: FormElement.inc [hunk 2]
```diff
@@ -2,10 +2,9 @@
 
 /**
  * @file
- *
- * Defines a FormElement class for encapsulating Drupal form elements. Each FormElement class has a unique #hash
- * property that can be used to identify that FormElement.
+ * Defines a FormElement class for encapsulating Drupal form elements. Each FormElement class has a unique #hash property that can be used to identify that FormElement.
```
Comment (jonathangreen) [inline FormElement.inc]: Same as above for the file comment.

File: FormElement.inc [hunk 3]
```diff
@@ -207,21 +213,19 @@ class FormElement implements ArrayAccess {
   }
 
   /**
-   * Takes a callback, and calls it repeatedly passing each control/property to the callback
-   * with the name of the control/property as the first parameter and the value of the control/property
-   * as the second paramter. Any other parameters passed to this function will be passed to the
-   * callback as additional parameters.
+   * Takes a callback, and calls it repeatedly passing each control/property to the callback with the name of the control/property as the first parameter and the value of the control/property as the second paramter. Any other parameters passed to this function will be passed to the callback as additional parameters.
```
Comment (jonathangreen) [inline FormElement.inc]: I'm thinking we could do the same thing here as with the file comments. We could add a little summary sentence. Then have the rest of the comment as a second paragraph. Some examples are here:
http://drupal.org/coding-standards/docs#functions

**Comment by jonathangreen** [comment]
This looks good other then the function and file comments I mentioned.

**Comment by christianselig** [comment]
The lines seemed arbitrarily split up, for example the first one was split at column 118, so it was violating the 80 characters per line restriction anyway, I just put them back on one line until it was decided what was to be done with them.

**Comment by christianselig** [comment]
Is that alright @jonathangreen?

**Pull request closed without merging**